### PR TITLE
add_license-control-center

### PR DESCRIPTION
**Pull request by yutarody**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/license-control-center.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  sha256 'fe1387d8897e0c2b07096a2a85d6d3e80fd5ef2a1fd91823dc640400f6abd441'
+
+  url 'http://download.steinberg.net/downloads/eLicenserControl.dmg'
+  name 'eLicenser Control Center'
+  homepage 'http://www.steinberg.net/en/company/technologies/elicenser.html'
+  license :gratis
+
+  installer manual: "eLicenser Control #{version}/eLicenserControlSetup.app"
+
+  uninstall delete: '/Applications/License Control Center.app'
```
Comment (vitorgalvao) [inline Casks/license-control-center.rb]: Is moving this app the only thing the installer does? That seems unlikely.

File: Casks/license-control-center.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+cask 'license-control-center' do
+  version '6.9.3.8191'
```
Comment (vitorgalvao) [inline Casks/license-control-center.rb]: Seeing as `url` doesn’t point to a specific version, this cask should use `version :latest` and `sha256 :no_check`. [See an example](https://github.com/caskroom/homebrew-cask/blob/f25b6babcd398abf48e33af3d887b2d00de1d661/Casks/kibako.rb).

File: Casks/license-control-center.rb (hunk 1 above)
Comment (yutarody) [inline Casks/license-control-center.rb]: This 'eLC-Uninstaller.app' are bundled. This seems to be the application of Applescript.
When you confirm this Applescript There was following code.

```
to getELCFileList()
	set path01 to "/Applications/License\\ Control\\ Center.app"
	set path02 to "/Library/Application\\ Support/Syncrosoft"
	set path03 to "/Library/Application\\ Support/eLicenser"
	set path04 to "/Library/Syncrosoft"
	set path05 to "/Library/Receipts/eLicenserControl.pkg"
	set path06 to "/Library/Receipts/SyncrosoftLicenseControl.pkg"
	set path07 to "/Library/Receipts/LCC.pkg"
	set path08 to "/Library/Receipts/NOS.pkg"
	set path09 to "/Library/Receipts/POS.pkg"
	set path10 to "/var/db/receipts/com.syncrosoft.*"
	set path11 to "/var/db/receipts/com.eLicenser.*"
	set path12 to POSIX path of (path to home folder) & "/Library/Preferences/com.Syncrosoft.LCC.plist"
	set path13 to POSIX path of (path to home folder) & "/Library/Saved\\ Application\\ State/com.Syncrosoft.LCC.savedState"
	
	set fileList to {path01, path02, path03, path04, path05, path06, path07, path08, path09, path10, path11, path12, path13}
	
	return fileList
end getELCFileList
```

I thought it is only necessary to remove the 'app'. 
Please give me advice.
 ( '/ Library / Receipts / *' This did not actually exist anything)

Comment (vitorgalvao) [inline Casks/license-control-center.rb]: Those `/Receipts/` might be good candidates to [`pkgutil`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/uninstall.md#uninstall-key-pkgutil).

All the others should be included in [`zap`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/zap.md).

Comment (yutarody) [inline Casks/license-control-center.rb]: Thank you for advice.
I was fixed some of the items.

File: Casks/license-control-center.rb (hunk 2 above)
Comment (yutarody) [inline Casks/license-control-center.rb]: Thank you.
I was fixed.

File: Casks/license-control-center.rb [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+  url 'http://download.steinberg.net/downloads/eLicenserControl.dmg'
+  name 'eLicenser Control Center'
+  homepage 'http://www.steinberg.net/en/company/technologies/elicenser.html'
+  license :gratis
+
+  installer manual: "#{staged_path}}/eLicenserControlSetup.app"
+
+  uninstall delete:  '/Applications/License Control Center.app',
+            pkgutil: [
```
Comment (vitorgalvao) [inline Casks/license-control-center.rb]: Not how `pkgutil` works. See the [scripts](https://github.com/caskroom/homebrew-cask/tree/master/developer/bin) and [documentation](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/uninstall.md) geared for that.

File: Casks/license-control-center.rb [hunk 4]
```diff
@@ -0,0 +1,29 @@
+cask 'license-control-center' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://download.steinberg.net/downloads/eLicenserControl.dmg'
+  name 'eLicenser Control Center'
+  homepage 'http://www.steinberg.net/en/company/technologies/elicenser.html'
+  license :gratis
+
+  installer manual: "#{staged_path}}/eLicenserControlSetup.app"
```
Comment (vitorgalvao) [inline Casks/license-control-center.rb]: There’s an extra `}`. Also `#{staged_path}` is unnecessary here.

Comment (yutarody) [inline Casks/license-control-center.rb]: Oh ... I'm sorry
I fix it

File: Casks/license-control-center.rb (hunk 3 above)
Comment (yutarody) [inline Casks/license-control-center.rb]: I was changed to "delete" from "pkgutil". 
Also I was referring to the following data.
[https://www.steinberg.net/en/support/knowledgebase_new/show_details/kb_show/how-to-uninstall-the-elicenser-control-centerlicense-control-center](https://www.steinberg.net/en/support/knowledgebase_new/show_details/kb_show/how-to-uninstall-the-elicenser-control-centerlicense-control-center)

Comment (vitorgalvao) [inline Casks/license-control-center.rb]: > Also I was referring to the following data.
> https://www.steinberg.net/en/support/knowledgebase_new/show_details/kb_show/how-to-uninstall-the-elicenser-control-centerlicense-control-center

Great! Having official data is always best.

**Pull request merged**